base/settings/R/MultiSettings.R

```diff
@@ -142,59 +148,61 @@ settingNames <- function(multiSettings, settingNames) {
   }
 } # settingNames
 
-##' @export
+#' @export
 print.MultiSettings <- function(x, printAll = FALSE, ...) {
   if (printAll) {
     NextMethod()
   } else {
-    print(paste0("A MultiSettings object containing ", length(x), " Settings."), ...)
+    print(
```

Comment:
do we want to switch this to a log statement?

Reply:
I'd say no -- the print method for a settings object can get called in quite a few settings where a log statement would be inappropriate, e.g. inspecting settings during an interactive session, or inside a `logger.info(print2string(settings))`.